plugin/session_view.py

```diff
@@ -301,22 +302,23 @@ def diagnostics_tag_scope(self, tag: int) -> str | None:
     def present_diagnostics_async(
         self, is_view_visible: bool, data_per_severity: dict[tuple[int, bool], DiagnosticSeverityData]
     ) -> None:
+        self._diagnostics_data_per_severity = data_per_severity
```

Comment:
Wouldn't it be more correct to store this in `SessionBuffer` since this data is the same for each SessionView belonging to the same SessionBuffer?

Reply:
https://github.com/sublimelsp/LSP/pull/2448/commits/b2bab2f146c06da56f19a80ac574c4c3ac223005